components/UI/SitesIcon/Icons.tsx

```diff
@@ -0,0 +1,140 @@
+import Avatar from "@mui/material/Avatar";
+import Stack from "@mui/material/Stack";
+import Popover from "@mui/material/Popover";
+import Box from "@mui/material/Box";
+import Typography from "@mui/material/Typography";
+
+import React from "react";
+
+const SitesIcon = () => {
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+  const [isOpen, setIsOpen] = React.useState<any>(null);
+  const handlePopoverOpen = (Popover: string) => {
```

Comment:
useRef() can solve this more optimal way? we should use react.js solutions

Reply:
fixed